addons/l10n_ar/data/l10n_latam.document.type.csv

```diff
@@ -53,16 +53,16 @@ dc_cfcp,510,71,CREDITO FISCAL POR CONTRIBUCIONES PATRONALES,,,,,base.ar,,False
 dc_f1116,520,73,FORMULARIO 1116 RT,,,,,base.ar,,False
 dc_cptag,530,74,CARTA DE PORTE PARA EL TRANSPORTE AUTOMOTOR PARA GRANOS,,,,,base.ar,,False
 dc_cptfg,540,75,CARTA DE PORTE PARA EL TRANSPORTE FERROVIARIO PARA GRANOS,,,,,base.ar,,False
-dc_zeta,550,80,COMPROBANTE DIARIO DE CIERRE (ZETA),,,,,base.ar,,False
-dc_a_t,560,81,TIQUE FACTURA A   CONTROLADORES FISCALES,A,TIQUE FACTURA,invoice,TF-A,base.ar,not_zero,True
+dc_zeta,550,80,INFORME DIARIO DE CIERRE (ZETA) - CONTROLADORES FISCALES,,,,,base.ar,,False
+dc_a_t,560,81,TIQUE FACTURA A   ,A,TIQUE FACTURA,invoice,TF-A,base.ar,not_zero,True
```

Comment:
@Bruno-Zanotti y would remove the spaces on `A   ` I know that afip table has this space, but better to remove it

Reply:
:+1: